src/configurations/destinations/http/db-config.json

```diff
@@ -51,7 +51,8 @@
         "maxBatchSize",
         "blacklistedEvents",
         "whitelistedEvents",
-        "eventFilteringOption"
+        "eventFilteringOption",
+        "isDefaultMapping"
```

Comment:
should we say, skipMapping?

Reply:
In backend when isDefaultMapping is enabled, we'll be using default mapping {to: "$", from: "$"}. So I named it this way